Test/AssemblyInfo.cs

```diff
@@ -0,0 +1,4 @@
+﻿using NUnit.Framework;
```

Comment:
I think you removed this in your previous pull request, why adding it back here?